### PR TITLE
Modified three tests to remove the "skip" to enable the tests to execute

### PR DESCRIPTION
**Pull request by mdlpstsci**
Remove the "skip" to enable the tests to execute.  PyTest criterion may have been modified to make the test succeed or be more general.

### CODE REVIEW

File: tests/hap/test_run_svmpoller.py [hunk 1]
```diff
@@ -60,29 +54,34 @@ def test_run_svmpoller(tmpdir, dataset):
...
     print("TEST_RUN_SVMPOLLER. Dataset: ", dataset)
 
     current_dt = datetime.datetime.now()
     print(str(current_dt))
 
+    # Create working directory specified for the test
+    #install_dir = os.path.dirname(__file__)
+    #local_path = os.path.join(install_dir, dataset)
+    #os.chdir(install_dir)
+
```
Comment (stsci-hack) [inline tests/hap/test_run_svmpoller.py:84]: Code that has been commented out can be simply deleted if it is not needed for the final working version.

**Review by stsci-hack — CHANGES_REQUESTED** [review]
Just a couple of things stood out to be addressed.

File: tests/hap/test_run_svmpoller.py [hunk 2]
```diff
@@ -3,55 +3,49 @@
...
-from astropy.table import Table
 
+from astropy.io import ascii
 from drizzlepac import runsinglehap
-
-
-log = logutil.create_logger('test_run_svmpoller', level=logutil.logging.INFO, stream=sys.stdout)
+from astropy.table import Table
+from drizzlepac.haputils import astroquery_utils as aqutils
+from importlib import reload
```
[the remark this answers is not in the document]
Comment (mdlpstsci) [reply tests/hap/test_run_svmpoller.py]: Fixed in e29c47d.

File: tests/hap/test_run_svmpoller.py (hunk 1 above)
Comment (mdlpstsci) [reply tests/hap/test_run_svmpoller.py]: Fixed in e29c47d.

**Review by stsci-hack — APPROVED** [review]
Looks fine to me now.

**Pull request merged**